src/components/Disputes/actions/DisputeAppeal.js

```diff
@@ -24,8 +24,22 @@ function DisputeAppeal({ onRequestAppeal, confirm }) {
       >
         {actionLabel}
       </Button>
-      <Info mode="description">
-        Anyone holding the required appeal deposit can trigger this action.
+      <Info>
+        <strong>Anyone</strong> can{' '}
+        <strong>
+          lock DAI as collateral to{' '}
+          {confirm ? 'confirm an appeal' : 'propose an appeal'}{' '}
+        </strong>
+        , if they don’t agree with the current outcome. When the final ruling is
+        confirmed, the user who {confirm ? 'confirmed the appeal' : 'appealed'}{' '}
+        gets rewarded if the ruling has switched in their favor. If not, their
+        entire collateral could be re-distributed to the winning party.{' '}
+        {confirm
+          ? 'If an appeal is confirmed, a new adjudication round is initiated and a new jury is drafted.'
+          : ''}
+        <Link href="https://help.aragon.org/article/43-dispute-lifecycle#appeal">
+          Learn more
```

Comment:
Let's add a period afterwards (we should always end the "Learn more"s with a period).